app/views/trainees/funding/bursaries/_non_tiered_bursary_form.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= register_form_with(model: @bursary_form, url: trainee_funding_bursary_path(@trainee), method: :put, local: true) do |f| %>
+  <%= f.govuk_error_summary %>
+
+  <h1 class="govuk-heading-l"><%= t("components.page_titles.trainees.funding.bursaries.edit") %></h1>
+  <p class="govuk-body">
+    <%= t(
+      (@trainee.early_years_route? ? "views.forms.funding.bursaries.non_tiered.early_years_description" : "views.forms.funding.bursaries.non_tiered.description"),
+      amount: format_currency(@amount),
+      subject: @subject,
+      training_route: t("activerecord.attributes.trainee.training_routes.#{@trainee.training_route}")
+    ) %>
+    </br>
```

Comment:
Realise this was how they were separated before, but I think that might have been because it was hiding in `en.yml`!

I think the html should be:

```
<h1></h1>
<p class="govuk-body">Description part</p>
<p class="govuk-body">Link to guidance part</p>
```